apps/backend/src/routes/signup/onboard/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<!-- JavaScript code for Continue button -->
+<script lang="ts">
+	let confirm: boolean = false;
+</script>
+
+<!-- Onboard Form - Logo -->
+<div class="skeleton w-32 h-32"></div>
+
+<div class="flex flex-row items-center justify-center h-screen">
+	<!-- Onboard Form - Card -->
+	<div class="w-full max-w-2xl p-10 border border-primary rounded-md shadow-lg">
+		<!-- Onboard Form - Header -->
+		<h1 class="text-3xl font-semibold text-center mb-5">Get Onboard</h1>
+
+		<h2 class="text-center">Complete your profile to start using Armadillo!</h2>
```

Comment:
This should be a `span` tag instead of `h2`.

Reply:
Changed